src/lib/sidebar/scheme_data.ts

```diff
@@ -22,6 +23,10 @@ export function backfill(json: Scheme) {
     // Always overwrite IDs, and follow what newFeatureId requires.
     f.id = idCounter++;
   }
```

Comment:
I think we need logic here to backfill the previous format we've been using. We might have `json.scheme_name` set. If so, we need to move it over to the default `schemes` thing that gets created